### PR TITLE
Update the tutorial with changes recommended in #2761

### PR DESCRIPTION
**Pull request by CameronAavik**
I've included fixes for each recommendation in #2761 except for the one I mentioned on line 111-112 since I realise now that the comment makes sense in the section is was already in and isn't required to be earlier in the code.

### CODE REVIEW

File: vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx [hunk 1]
```diff
@@ -67,6 +67,9 @@ module IntegersAndNumbers =
     // The next line prints a list that includes tuples, using '%A' for generic printing.
     printfn "The table of squares from 0 to 99 is:\n%A" sampleTableOfSquares
 
+    // This is a sample integer with a type annotation
+    let sampleInteger3: int = 1;
```
Comment (vasily-kirichenko) [inline vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx]: Please, remove the trailing semicolon.

File: vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx [hunk 2]
```diff
@@ -201,6 +205,10 @@ module Tuples =
     let tuple2 = (1, "fred", 3.1415)
 
     printfn "tuple1: %A\ttuple2: %A" tuple1 tuple2
+    
+    /// A simple tuple of integers with a type annotation.
+    /// Type annotations for tuples use the * symbol to separate elements
+    let tuple3: (int*int) = (5, 9)
```
Comment (vasily-kirichenko) [inline vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx]: We normally do not surround type annotations with parens, and not always place parens around tuple values, so `let tuple3 : int * int = 5, 9` is a more idiomatic code.

File: vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx [hunk 3]
```diff
@@ -330,7 +338,7 @@ module Lists =
     let daysList = 
         [ for month in 1 .. 12 do
               for day in 1 .. System.DateTime.DaysInMonth(2017, month) do 
-                  yield System.DateTime(2012, month, day) ]
+                  yield System.DateTime(2017, month, day) ]
```
Comment (vasily-kirichenko) [inline vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx]: ;)

**Comment by CameronAavik** [comment]
Do you prefer to have those modifications squashed? Or are you fine with them as separate commits

**Comment by forki** [comment]
Separate commits is fine. And thanks a lot for doing this.

**Comment by vasily-kirichenko** [comment]
The maintainers always squash PRs into one commit anyway, so don't worry about it.

File: vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx [hunk 4]
```diff
@@ -42,8 +42,8 @@
 open System
 
 
-/// Modules are the primary way to organize functions and values in F#. This module contains some
-/// basic values involving basic numeric values computed in a few different ways.
+/// A module is a grouping of F# code, such as values, types, and function values, in an F# program.
```
Comment (cartermp) [inline vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx]: Nit: I'd remove "in an F# program."  It's already stated at the beginning of the sentence that we're dealing with F# code.

**Comment by CameronAavik** [comment]
I have no clue why that build failed. All I did was remove a couple words in a comment haha.

**Comment by cartermp** [comment]
No worries @CameronAavik - looks like an issue with a package not being found.

**Comment by cartermp** [comment]
@dotnet-bot test this please

**Comment by CameronAavik** [comment]
Hmmm, although I doubt it's my change that caused it, the issue has occurred twice in a row with the same symptom, seemingly only on my pull request. Is someone able to figure out what's wrong with it?

File: vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx [hunk 5]
```diff
@@ -67,6 +67,9 @@ module IntegersAndNumbers =
     // The next line prints a list that includes tuples, using '%A' for generic printing.
     printfn "The table of squares from 0 to 99 is:\n%A" sampleTableOfSquares
 
+    // This is a sample integer with a type annotation
```
Comment (KevinRansom) [inline vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx]: @CameronAavik 

Don't worry about squashing, I will squash when I pull the PR.

do you think that the sample with annotation should be closer to the sample integer without annotation.  It seems that would that make it more relevant?

File: vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx [hunk 6]
```diff
@@ -116,7 +120,7 @@ module BasicFunctions =
     let sampleFunction2 (x:int) = 2*x*x - x/5 + 3
 
     let result2 = sampleFunction2 (7 + 4)
-    printfn "The result of applying the 1st sample function to (7 + 4) is %d" result2
```
Comment (KevinRansom) [inline vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx]: perhaps:   "The result of applying sampleFunction2 to 

and ditto for other examples ...

File: vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx (hunk 3 above)
Comment (KevinRansom) [inline vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx]: Yes we should definitely modernize the samples ... I like it :-)

**Comment by CameronAavik** [comment]
Hey Kevin, I'm at work at the moment, but I can make these changes when I get home later tonight.

**Comment by KevinRansom** [comment]
@CameronAavik 
In your own time my friend.  There is no rush ... we've been misspelling stuff for years :-)

**Comment by KevinRansom** [comment]
@CameronAavik 

Submit it in another PR.  I will pull this one

And thanks for taking care of this stuff.

Kevin

**Pull request merged**